ChatQnA/README.md

```diff
@@ -49,6 +52,41 @@ You have the flexibility to customize these parameters according to your specifi
 export TGI_LLM_ENDPOINT="http://xxx.xxx.xxx.xxx:8080"
 ```
 
+## Environment Setup for LLM-on-Ray
+To use Intel [LLM-on-Ray](https://github.com/intel/llm-on-ray) on Habana Gaudi/Gaudi2, please follow these steps:
+
+### Launch Ray Gaudi Service
+
+#### Launch a local server instance on 1 Gaudi card:
+```bash
+bash ./serving/ray_gaudi/launch_ray_service.sh
+```
+
+For gated models such as `LLAMA-2`, you will have to pass -e HUGGING_FACE_HUB_TOKEN=\<token\> to the docker run command above with a valid Hugging Face Hub read token.
+
+Please follow this link [huggingface token](https://huggingface.co/docs/hub/security-tokens) to get the access token ans export `HUGGINGFACEHUB_API_TOKEN` environment with the token.
+
+```bash
+export HUGGINGFACEHUB_API_TOKEN=<token>
+```
+
+#### Launch a local server instance on 8 Gaudi cards:
+```bash
+bash ./serving/ray_gaudi/launch_ray_service.sh 8
+```
+
+#### Customize Ray Gaudi Service
+
+The ./serving/ray_gaudi/launch_ray_service.sh script accepts three parameters:
+- num_cards: The number of Gaudi cards to be utilized, ranging from 1 to 8. The default is set to 1.
+- port_number: The port number assigned to the Ray Gaudi endpoint, with the default being 8080.
+- model_name: The model name utilized for LLM, with the default set to "Intel/neural-chat-7b-v3-3".
```

Comment:
We don't have to follow the script for TGI. What about directly asking user to update the model yaml file in llm-on-ray which will be more flexible? In the document, we don't need to explain every parameter. We can explain the default yaml file is for a single model replica with 1 card, and explain how to update if we want to use 8 cards, or another model.